ocular/src/chain/client.rs

```diff
@@ -9,6 +9,7 @@ use futures::executor;
 use tendermint_rpc;
 
 pub mod query;
+pub mod transactions;
```

Comment:
Nit: I think we should go with `tx` instead of `transactions`, here and anywhere else the word is referenced (`TxError`, `TxMetadata`)

Reply:
That sounds cleaner! Will update.

Reply:
Updated.